### ISSUE TITLE
Look into Author fields post-article2

### ISSUE THREAD

**Issue by eessex** [artsy/positron#1299]
The Primary author field saves the Name and id of the channel in which an article was created. 

For all articles, we currently show an editable field to change the Primary Author name in the Admin Panel.  However, this will not be rendered in the future for editorial articles. 

Post-new article launch, we can look into removing the UI for this from editorial channels -- where the new author_ids field is used instead for display. 

We will also be deprecating the contributing authors field, so we should evaluate what steps are necessary before removing it from the editorial UI.

**Comment by kanaabe** [comment]
> The Primary author field saves the Name and id of the channel in which an article was created.

Can you double check that it is the channel id? I'm pretty sure the id is mapping to the user who created the article and the name just gets overwritten but i may be wrong.

> We will also be deprecating the contributing authors field, so we should evaluate what steps are necessary before removing it from the editorial UI.

Contributing authors will have to stay to support classic layouts since new Authors must be added manually into Positron's db instead of pulling a Gravity user.

It'd be helpful to do a search across all distributions and in Writer (both API and client) for places we might still use this field

**Comment by eessex** [comment]
Contributing authors would be deprecated for the editorial channels only- I think we should remove the ui for those channels if it is not being rendered.

**Comment by kanaabe** [comment]
I agree we should remove editorial UI -- I just wanted to make sure the API expectation was clear!
